<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hashing and AES/RSA Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>611</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_hash_file">
     <attribute name="title">
      <string>Hasher un fichier</string>
     </attribute>
     <widget class="QLabel" name="label_hash_input_file">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sélectionner fichier à hacher :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_hash_input_file">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_hash_input">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Parcourir</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_hash_output_file">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_generate_hash">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Générer Hash</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_hash_input_file_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>HASH 256 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_hash_input_hash">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_hash_input_file_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_hash_input_file_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>HASH 256 :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_test_hash">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>290</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>verify</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_hash_input_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>190</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_hash_input_file_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sélectionner fichier à hacher :</string>
      </property>
     </widget>
     <widget class="QLabel" name="HashVerif">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>300</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_aes">
     <attribute name="title">
      <string>AES Encryption/Decryption Tool</string>
     </attribute>
     <widget class="QLabel" name="label_input_file">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>150</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Input File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_input_file">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>240</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_input">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_output_file">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>150</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Output File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_output_file">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>240</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_output">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_aes_key">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>150</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Save AES Key As:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_aes_key">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>240</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_aes_key">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>130</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_encrypt">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_decrypt_input_file">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>150</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Encrypted File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_decrypt_input_file">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>240</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_decrypt_input">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>230</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_decrypt_output_file">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>150</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Output File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_decrypt_output_file">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>240</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_decrypt_output">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>280</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_load_aes_key">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>150</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Load AES Key:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_load_aes_key">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>330</y>
        <width>240</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_load_aes_key">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>330</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_decrypt">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>380</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_generate_key">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>550</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>generate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_input_aes_key">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>500</y>
        <width>240</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_file_to_key">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>500</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_load_aes_key_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>500</y>
        <width>150</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path to save key:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_rsa">
     <attribute name="title">
      <string>RSA Tool</string>
     </attribute>
     <widget class="QLabel" name="label_rsa_key_save_pub">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Public Key As:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rsa_pub_key">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_rsa_pub">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_rsa_key_save_priv">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Private Key As:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rsa_priv_key">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_rsa_priv">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_generate_rsa_keys">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate RSA Keys</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_rsa_encrypt_input_file">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Select File to Encrypt:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rsa_encrypt_input_file">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_rsa_encrypt_input">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>240</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_rsa_encrypt_output_file">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Encrypted Output File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rsa_encrypt_output_file">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>280</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_rsa_encrypt_output">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>280</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rsa_encrypt">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>320</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt with Public Key</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_rsa_decrypt_input_file">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Encrypted File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rsa_decrypt_input_file">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>450</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_rsa_decrypt_input">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>450</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_rsa_decrypt_output_file">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Decrypted Output File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rsa_decrypt_output_file">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>490</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_rsa_decrypt_output">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>490</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rsa_decrypt">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>530</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt with Private Key</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_rsa_decrypt_input_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>410</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Private_Path">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>410</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_rsa_decrypt_input_file_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Select  File to private key:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_rsa_public_encrypt">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>200</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rsa_public_path">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_rsa_encrypt_input_file_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Select File to public key:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
